package.json

```diff
@@ -4,7 +4,8 @@
   "description": "The Core functionality of Doctor - The Cloud Elements Asset Management Tool",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "jest",
+    "test:coverage": "jest --coverage"
```

Comment:
Add prettier as well, `"prettier": "./node_modules/.bin/prettier --write --print-width=120 --tab-width=2 --single-quote=true --trailing-comma=all --bracket-spacing=false src"`

Reply:
@MohdRafiUddin Will open a separate PR for lint/prettifier as it will cause changes in almost all files